### DISCUSSION TITLE
[How to] Customizable homepage tutorial.

### DISCUSSION THREAD

**Discussion by A3Bagged** [squidfunk/mkdocs-material#7838] [Show and tell]
Hello 👋  since there were many people wondering about homepages _(including myself at first)_
i've made a quick documentation on how to create these custom homepages.

[you can find it here](https://a3bagged.github.io/theme-test/custom-homepage/)

if there are any questions or issues/misspellings do not hestitate to [contact](https://github.com/A3Bagged/theme-test/discussions).

<a href="https://a3bagged.github.io/theme-test/homepage-live-demo/"><img src="https://github.com/user-attachments/assets/396dba0f-6fde-4b86-ae68-7e35eae5d6d8"></a>

Enjoy! 😎

**Comment by Pengyu-Jin** [comment]
Thanks for sharing this great tutorial. But to me, don't know too much about the frontend technology, it's still a little difficult to understand all the settings😑, all in all,  thanks.🥰

**Comment by shenweiyan** [comment]
Cool job 👍
